packages/unity-react-core/src/core/utils/index.js

```diff
@@ -130,3 +130,9 @@ export {
   initImageCarousel,
   initImageGalleryCarousel,
 };
+import { SidebarMenu } from "../../components/SidebarMenu/SidebarMenu.tsx";
+
+ // @param {ComponentProps} props
+export const initSidebarMenu = ({ targetSelector, props }) =>
```

Comment:
All other exports are arranged in alpha order, let's keep to that pattern with this export, too, please.